repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -20,42 +20,69 @@
 
 package org.openrepose.filters.samlpolicy
 
+import java.io.{FileInputStream, StringReader}
+import java.security.cert.X509Certificate
+import java.security.{KeyStore, Security}
 import java.util.Base64
-import java.io.StringReader
-import javax.servlet.FilterChain
 import javax.servlet.http.HttpServletResponse.SC_BAD_REQUEST
 import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import javax.servlet.{FilterChain, FilterConfig}
 import javax.xml.parsers.DocumentBuilderFactory
 import javax.xml.transform.stream.StreamSource
 
+import com.typesafe.scalalogging.slf4j.LazyLogging
 import net.sf.saxon.s9api.Processor
+import net.shibboleth.utilities.java.support.resolver.CriteriaSet
 import org.junit.runner.RunWith
 import org.mockito.Mockito._
 import org.mockito.{Matchers => MM}
 import org.openrepose.core.services.config.ConfigurationService
 import org.openrepose.core.services.serviceclient.akka.AkkaServiceClientFactory
-import org.openrepose.filters.samlpolicy.config.{Cache, PolicyAcquisition, SamlPolicyConfig}
+import org.openrepose.filters.samlpolicy.config.{Cache, PolicyAcquisition, SamlPolicyConfig, SignatureCredentials}
 import org.openrepose.nodeservice.atomfeed.AtomFeedService
+import org.opensaml.core.config.{InitializationException, InitializationService}
+import org.opensaml.core.criterion.EntityIdCriterion
+import org.opensaml.saml.saml2.core.impl.ResponseUnmarshaller
+import org.opensaml.security.credential.CredentialSupport
+import org.opensaml.security.credential.impl.StaticCredentialResolver
+import org.opensaml.xmlsec.config.JavaCryptoValidationInitializer
+import org.opensaml.xmlsec.keyinfo.impl.StaticKeyInfoCredentialResolver
+import org.opensaml.xmlsec.signature.SignableXMLObject
+import org.opensaml.xmlsec.signature.support.impl.ExplicitKeySignatureTrustEngine
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.mock.MockitoSugar
 import org.scalatest.{BeforeAndAfterEach, FunSpec, Matchers}
+import org.slf4j.{Logger, LoggerFactory}
 import org.springframework.test.util.ReflectionTestUtils
-import org.xml.sax.InputSource
+import org.w3c.dom.Document
 
 import scala.io.Source
+import scala.xml.InputSource
 
 /**
   * Created by adrian on 12/14/16.
   */
 @RunWith(classOf[JUnitRunner])
-class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach with Matchers with MockitoSugar {
+class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach with Matchers with MockitoSugar with LazyLogging {
```

Comment:
This is part of the original OpenSAML testing. I updated it to the local logger in the static object method that uses it. It indicates if there is an issue with the JCE environment being used by the library.